controllers/bootstrap/init.go

```diff
@@ -111,6 +111,21 @@ func (b *Bootstrap) InitResources(manualManagement bool) error {
 		return err
 	}
 
+	// Check Saas Deployment
+	controlNs := b.MasterNamespace
+	saasEnable, err := util.CheckSaas(b.Reader)
+	if err != nil {
+		return err
+	}
+	if saasEnable {
```

Comment:
we should have additional check here for `common-service-maps` to support installing multiple CS instances in non-SaaS environment

```suggestion
	}
	
	// check multiple CS instances installation
	isMultiCs, err := util.IsMultiCs(b.Reader)
	if err != nil {
	    return err
	}
	
	if saasEnable || isMultiCs{
```

Reply:
Based on slack message, the difference of SaaS and MultiCS is `OperandRegistry` and `OperandConfig`, does it indicate that we will deploy cluster scope resources in cs control namespace for both cases?

Reply:
If we do not deploy in control namespace, how would multiple CS work?

Reply:
But there is a problem, I think that it will introduce a shared control namespace even for single instance installation.
How do we know it is a multiple CS deployment in non-SaaS?

From my understanding, so far in the non-SaaS deployment, the `common-services-maps` is created by user manually.

Firstly, they create the ConfigMap with only one instance of CS like this
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: common-service-maps
  namespace: kube-public
data: 
  common-service-maps.yaml: |
    namespaceMapping:
    - requested-from-namespace:
      - cp4i
      - cp4mcm
      map-to-common-service-namespace: cp-common-service
```

After the installation of CS, they add another mapping into the ConfigMap and start another installation.
apiVersion: v1
kind: ConfigMap
metadata:
  name: common-service-maps
  namespace: kube-public
data: 
  common-service-maps.yaml: |
    namespaceMapping:
    - requested-from-namespace:
      - cp4i
      - cp4mcm
      map-to-common-service-namespace: cp-common-service
     - requested-from-namespace:
      - iaf
      - watson
      map-to-common-service-namespace: watson-common-service
```

So at the first stage, the common service knows that there is only one instance, and deploy everything(current case) or into one namespace called `cp-common-service`. Or deploy cluster-scope resources into shared namespace(new design).

After the user add another mapping into ConfigMap and start the installation of second instance of CS, the CS will recognize that it is a multiple instance case.
But at this time some cluster scope resources have been deployed in the first namespace.

If we have a shared namespace, everything goes fine.

If we don't have a shared namespace for cluster scope resources in Non-SaaS multi-instance deployment.
We need to use the first CS-deployed namespace as the master namespace for the cluster scope resources.

/cc @horis233

Reply:
update to check the SaaS and multi instances deployment by `control namespace`